internal/transform/config_setting.go

```diff
@@ -0,0 +1,161 @@
+package transform
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/stellar/go/ingest"
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+// TransformConfigSetting converts an config setting ledger change entry into a form suitable for BigQuery
+func TransformConfigSetting(ledgerChange ingest.Change) (ConfigSettingOutput, error) {
+	ledgerEntry, changeType, outputDeleted, err := utils.ExtractEntryFromChange(ledgerChange)
+	if err != nil {
+		return ConfigSettingOutput{}, err
+	}
+
+	configSetting, ok := ledgerEntry.Data.GetConfigSetting()
+	if !ok {
+		return ConfigSettingOutput{}, fmt.Errorf("Could not extract config setting from ledger entry; actual type is %s", ledgerEntry.Data.Type)
+	}
+
+	// LedgerEntryChange must contain a config setting state change to be parsed, otherwise skip
+	if ledgerEntry.Data.Type != xdr.LedgerEntryTypeConfigSetting {
+		return ConfigSettingOutput{}, nil
+	}
```

Comment:
Is this redundant given the `GetConfigSetting()` method? curious if this will ever trigger